week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,9 @@
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var totalOfPeople = numberOfStudents + numberOfMentors;
+var perOfStudents = (numberOfStudents / totalOfPeople) * 100;
+var perOfMentors = (numberOfMentors / totalOfPeople) * 100;
+var perRooughMentors = Math.round(perOfMentors);
+var perRoughStudents = Math.round(perOfStudents);
+console.log("Persentage students: " + perRoughStudents + "%");
+console.log("Persentage mentors: " + perRooughMentors + "%");
```

Comment:
ah looks like it will do the calculation, but can u try with using 2 functions to get the result? - i.e. "You should have one function that calculates the percentage, and one function that creates a message"

Reply:
yes, i dont know why didnt i do it.